src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,144 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        displayWelcomeMsg();
+
+        Task[] array = new Task[100];
```

Comment:
You can use ArrayList in Java instead so you wouldn't have to declare a fixed amount of task. What happens if user enters more than 100 tasks?